doc/pages/Install.md

```diff
@@ -40,6 +40,8 @@ We provide pre-compiled binaries for:
 - OSX (intel 64 bits)
 (other platforms are available using the other methods below)
 
+Note: We are looking for help packaging Opam for Windows.
+
```

Comment:
We're not "looking" in the sense that no one is doing it at the moment: the core team's goal is merging (in terms of requirements) with https://fdopen.github.io/opam-repository-mingw/ (which is a binary packaging of opam).

Perhaps:
```suggestion
- We do not at present provide an official Windows distribution of opam, but please see [this separately maintained distribution](https://fdopen.github.io/opam-repository-mingw/)
(other platforms are available using the other methods below)

```